pkg/management/component.go

```diff
@@ -1119,40 +1114,22 @@ func (c *component) GetCredentialsForUser(ctx context.Context, realmName string,
 func (c *component) DeleteCredentialsForUser(ctx context.Context, realmName string, userID string, credentialID string) error {
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
-	// get the list of credentails of the user
-	credsKc, err := c.keycloakClient.GetCredentials(accessToken, realmName, userID)
+	// Check that credential is removable
+	var err = keycloakb.CheckRemovableMFA(ctx, credentialID, func() ([]kc.CredentialRepresentation, error) {
+		return c.keycloakClient.GetCredentials(accessToken, realmName, userID)
+	}, c.logger)
 	if err != nil {
-		c.logger.Warn(ctx, "msg", "Could not obtain list of credentials", "err", err.Error())
 		return err
 	}
 
-	// Ensure the credential is owned by user
```

Comment:
Ensure the credential is owned by user should remains.

Reply:
What is the use case?